src/coaches/coaches.controller.ts

```
@@ -0,0 +1,23 @@
+import { Controller, Get, Param, ParseUUIDPipe } from '@nestjs/common';
+import { UUID } from 'crypto';
+import { CoachesService } from './coaches.service';
+
+@Controller('coaches')
+export class CoachesController {
+    constructor(private readonly coachesService: CoachesService) {}
+    
+    // @UseInterceptors(LoggingInterceptor)
+    @Get()
```

ESLint

[error] Delete `··`

(prettier/prettier)